packages/loot-core/src/client/actions/account.ts

```diff
@@ -97,16 +97,64 @@ export function linkAccountSimpleFin(
   };
 }
 
+function handleSyncResponse(accountId, res, dispatch) {
+  const { errors, newTransactions, matchedTransactions, updatedAccounts } = res;
+
+  // Mark the account as failed or succeeded (depending on sync output)
+  const [error] = errors;
+  if (error) {
+    // We only want to mark the account as having problem if it
+    // was a real syncing error.
+    if (error.type === 'SyncError') {
+      dispatch(markAccountFailed(accountId, error.category, error.code));
+    }
+  } else {
+    dispatch(markAccountSuccess(accountId));
+  }
+
+  // Dispatch errors (if any)
+  errors.forEach(error => {
+    if (error.type === 'SyncError') {
+      dispatch(
+        addNotification({
+          type: 'error',
+          message: error.message,
+        }),
+      );
+    } else {
+      dispatch(
+        addNotification({
+          type: 'error',
+          message: error.message,
+          internal: error.internal,
+        }),
+      );
+    }
+  });
+
+  // Set new transactions
+  dispatch({
+    type: constants.SET_NEW_TRANSACTIONS,
+    newTransactions,
+    matchedTransactions,
+    updatedAccounts,
+  });
+
```

Comment:
This needs moved out, to avoid being called more than once in a single sync operation.  That causes the accounts/transactions to not bold properly.

Reply:
Done, could you give it a go please?

Reply:
@matt-fidd After fixing my broken account, did another update and things seem to work well in terms of bolding the accounts/transactions. So this looks fixed now.